### PR TITLE
Add basic scaffold for operator tests

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Changes proposed in this pull request:

- Add basic scaffold for operator tests

**Related issue(s)**

See also: https://github.com/kyma-project/keda-manager/issues/4

### CODE REVIEW

File: operator/controllers/keda_controller_test.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      kedaName,
+					Namespace: namespaceName,
+				},
+				Spec: v1alpha1.KedaSpec{},
+			}
+
+			Expect(k8sClient.Create(ctx, &keda)).Should(Succeed())
+
+			var sa corev1.ServiceAccount
```
Comment (anoipm) [inline operator/controllers/keda_controller_test.go:55]: Now I see what is `sa`, but I think that short names like this are bad idea. We save few letters and waste reader time.

**Review by anoipm — COMMENTED** [review]
Generally it's okay. I'm just leaving little comments on the variable naming convention.

File: operator/controllers/keda_controller_test.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Keda controller", func() {
+	Context("When creating fresh instance", func() {
+		const (
+			namespaceName = "keda"
+			kedaName      = "test"
+			saName        = "keda-manager"
```
[the remark this answers is not in the document]
Comment (m00g3n) [reply operator/controllers/keda_controller_test.go:25]: serviceAccountName, I will change it

**Review by anoipm — APPROVED** [review]

**Review by anoipm — APPROVED** [review]

**Pull request merged**